### PR TITLE
Multi-versionize py3-platformdirs

### PR DESCRIPTION
**Pull request by justinvreeland**
Build py-platformdirs for multiple python versions

### CODE REVIEW

File: py3-platformdirs.yaml [hunk 1]
```diff
@@ -32,13 +39,24 @@ pipeline:
...
+        with:
+          python: python${{range.key}}
+      - uses: strip
+    test:
+      pipeline:
+        - uses: python/import
+          with:
+            python: python${{range.key}}
+            import: ${{vars.pypi-packagename}}
+        - uses: strip
```
Comment (justinvreeland) [inline py3-platformdirs.yaml:59]: Opps need to remove this strip

Comment (smoser) [reply py3-platformdirs.yaml:59]: it doesnt hurt. and if this built any c-python bits you'd want it, right?

Comment (justinvreeland) [reply py3-platformdirs.yaml:59]: > it doesnt hurt. and if this built any c-python bits you'd want it, right?

It doesn't hurt. But this strip is on a test block there's already a strip on the building pipeline.

Comment (smoser) [reply py3-platformdirs.yaml:59]: ah. funny. yes, fix.

**Review by smoser — DISMISSED** [review]

File: py3-platformdirs.yaml [hunk 2]
```diff
@@ -32,13 +39,25 @@ pipeline:
...
-
-  - uses: strip
+subpackages:
+  - range: py-versions
+    name: py${{range.key}}-${{vars.pypi-packagename}}
+    description: ${{vars.pypi-package}} for python${{range.key}}
+    dependencies:
+      provider-priority: ${{range.value}}
+      provides:
+        - py3-${{pypi-packagename}}
```
Comment (jonjohnsonjr) [inline py3-platformdirs.yaml:49]:
```suggestion
        - py3-${{vars.pypi-packagename}}
```

**Review by smoser — APPROVED** [review]

**Pull request merged**